client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -140,10 +140,13 @@ enum EditorActions {
 	TrackPerformance = 'trackPerformance',
 }
 
-class CalypsoifyIframe extends Component<
-	Props & ConnectedProps & ProtectedFormProps & LocalizeProps & PerformanceTrackProps,
-	State
-> {
+type ComponentProps = Props &
+	ConnectedProps &
+	ProtectedFormProps &
+	LocalizeProps &
+	PerformanceTrackProps;
```

Comment:
When looking at code changes file in my IDE, I get a TypeScript error on this line:

`'"calypso/lib/performance-tracking"' has no exported member named 'PerformanceTrackProps'.`

Reply:
good idea 👍